assets/js/push.js

```diff
@@ -602,6 +607,10 @@ jQuery( window ).on( 'load', () => {
 				selectedConnections[ type + id ] = dtConnections[ type + id ];
 
 				const element = event.currentTarget.cloneNode( true );
+				document.querySelector(
+					'.selected-connections-count'
+				).textContent =
+					'( ' + Object.keys( selectedConnections ).length + ' )';
```

Comment:
```suggestion
					'(' + Object.keys( selectedConnections ).length + ')';
```

Consistent presentation

Reply:
Also I have used same as above format but still ESlint error is coming not sure why and also I have used same extension which you have provided for VS Code.

@peterwilsoncc There is one ESlint error cooming can you please check.

<img width="877" alt="image" src="https://user-images.githubusercontent.com/27722065/222123278-d581d275-25c7-42b8-ae67-57fb06fe7c3a.png">

Reply:
I see you figured this out. 

Prettier often applies different formatting depending on the length of the line. I've never been able to figure out what the logic is.